addons/api/addon/handlers/client-daemon-handler.js

```diff
@@ -75,14 +76,48 @@ const ClientDaemonHandler = {
           'query',
         );
 
-        // TODO: Add some pagination
+        // Return the raw data if we don't push to the store.
```

Comment:
This unfortunately means we wouldn't be returning models, do we feel OK with that?

Reply:
I don't exactly understand why we need the ArrayProxy here. Could you explain that further please?

Reply:
Is this related to my original comment about returning models or `ArrayProxy` only? Do you mean why do we need `ArrayProxy` at all?

Reply:
Yes, why do we need it at all. I read the code comments below and still wasn't exactly sure.
Also not a fan of not getting the model back, however if we know that going in then it's not a huge deal.

Reply:
It's to return a meta object, basically to imitate [this](https://guides.emberjs.com/release/models/handling-metadata/) so we have a way to return the total number of items in the result set when normally it would just be an array of the paginated data.

Reply:
Thanks! That page was helpful.